.github/dependabot.yml

```diff
@@ -3,33 +3,7 @@ updates:
 
   - package-ecosystem: "github-actions"
     directory: "/"
-    schedule:
-      interval: "weekly"
-    assignees:
-      - "@cartoncloud/code-owners-platform"
-    reviewers:
-      - "@cartoncloud/code-owners-platform"
-
-  - package-ecosystem: "github-actions"
-    directory: "/deployment-status"
-    schedule:
-      interval: "weekly"
-    assignees:
-      - "@cartoncloud/code-owners-platform"
-    reviewers:
-      - "@cartoncloud/code-owners-platform"
-
-  - package-ecosystem: "github-actions"
-    directory: "/jira-deployment-status"
-    schedule:
-      interval: "weekly"
-    assignees:
-      - "@cartoncloud/code-owners-platform"
-    reviewers:
-      - "@cartoncloud/code-owners-platform"
-
-  - package-ecosystem: "github-actions"
-    directory: "/slack-deployment-status"
+    open-pull-requests-limit: 1 # Limit PR's to 1 to avoid overwhelming
```

Comment:
Although I haven't seen the scenario here, I have experienced in java when two dependabot changes were both simulataneously required, and whilst I could have manually resolved them, it was just a bit easier/helpful to have the two appear at once and merge them together.  Not saying 1 is a bad number, just saying any related changes might not automatically be prepared

Reply:
Fair enough, I see your point. I was trying to reduce the number of PRs I get assigned to because it has caused me to miss some actually developer PRs. 

I will close this PR.